src/app/loadout/loadout-menu/LoadoutPopup.tsx

```diff
@@ -115,46 +118,6 @@ export default function LoadoutPopup({
     dispatch(applyLoadout(dimStore, loadout, { allowUndo: true }));
   };
 
-  const [confirmDialog, confirm] = useConfirm();
-  const applyRandomLoadout = async (e: React.MouseEvent, weaponsOnly = false) => {
-    e.stopPropagation();
-    if (
-      !(await confirm(
-        weaponsOnly ? (
-          t('Loadouts.RandomizeWeapons')
-        ) : query.length > 0 ? (
-          t('Loadouts.RandomizeSearchPrompt', { query })
-        ) : (
-          <>
-            {t('Loadouts.RandomizePrompt')}
-            <p className={styles.hint}>
-              {t('Loadouts.RandomizeQueryHint')}{' '}
-              <HelpLink helpLink={userGuideUrl('Randomize-Loadout')} />
-            </p>
-          </>
-        ),
-        { okLabel: t('Loadouts.RandomizeButton') }
-      ))
-    ) {
-      e.preventDefault();
-      onClick?.(e);
-      return;
-    }
-    try {
-      const loadout = randomLoadout(
-        dimStore,
-        allItems,
-        weaponsOnly ? (i) => i.bucket?.sort === 'Weapons' && searchFilter(i) : searchFilter
-      );
-      if (loadout) {
-        dispatch(applyLoadout(dimStore, loadout, { allowUndo: true }));
-      }
-    } catch (e) {
-      showNotification({ type: 'warning', title: t('Loadouts.Random'), body: errorMessage(e) });
-    }
-    onClick?.(e);
```

Comment:
I think this line made the loadout menu close after the randomize dialog finishes - probably still want that.